### ISSUE TITLE
Ship pfe-markdown with typography styles

### ISSUE THREAD

**Issue by starryeyez024** [patternfly/patternfly-elements#1701]
Currently the pfe-markdown component does not include baseline heading styles...  it just converts markdown to markup. But once it becomes markup, shouldn't it match the design system? If a page-builder adds:

```
<pfe-markdown >
  <h3> hello world</h3>
</pfe-markdown>
``` 
the H3 doesn't pick up the styles unless pfe-base.css is also loaded on the page.  Needing to load another stylesheet is a different requirement than most PFE components.
![image](https://user-images.githubusercontent.com/456863/124773940-63fb9b00-df0b-11eb-87e7-c271070983e5.png)

We *could* make the styles opt in, so that the PFE typographical styles are only applied if the page-builder adds the `pfe-c-content` class on it (or some other attribute) it can pick up the standard styles. Though it's also worth asking if the default styles should be opt-out. 🤔

```
<pfe-markdown class="pfe-c-content">
  <h3> hello world</h3>
</pfe-markdown>
``` 

Basically add lines 12-35 to the sass file: 
![image](https://user-images.githubusercontent.com/456863/124774144-92797600-df0b-11eb-9ed7-3b368d8b4775.png)
which results in 1KB minified, gzipped CSS.

**Comment by castastrophe** [comment]
> Needing to load another stylesheet is a different requirement than most PFE components.

The following components also require external stylesheets:
- pfe-band
- pfe-card
- pfe-tab-panel
- pfe-accordion-panel
- pfe-datetime / pfe-number
- pfe-jump-links-panel
- pfe-modal

There might be others as well that I'm not seeing.  Looking at that list, I would say a majority of our components have at least one region that does not provide any styling.  Might be worth a discussion on the team level if that is desired or if we want to update _all_ components to ship some amount of base styling.

**Comment by bennypowers** [comment]
Since these are all light-dom elements, the surrounding css context should apply. if the problem described is primarily taking place on CMS pages, can we solve it at that level by ensuring styles are applied from without?

Alternately, we may prefer to limit the use of (js-requiring) pfe-markdown in favour of server-side markdown rendering on CMS pages.

**Comment by bennypowers** [comment]
It appears this issue is no longer relevant. Please open a new issue if it is still relevant.
